src/libraries/System.Net.Security/src/System/Net/CertificateValidationPal.Android.cs

```diff
@@ -91,6 +91,10 @@ internal static partial class CertificateValidationPal
             return cert;
         }
 
+        // This is only called when we selected local client certificate.
+        // Currently this is only when Java crypto asked for it.
+        internal static bool IsLocalCertificateUsed(SafeFreeCredentials? _1, SafeDeleteContext? _2) => true;
```

Comment:
Why are these parameters named like this?

Reply:
Its because the parameters are unused. They are like this in main as well.

Reply:
Thanks. Personally, I'd rather we just suppress the analyzer that's leading us to this; _1 and _2 are strictly worse than their actual names. But if it's like this in main as well, fine, we can address it later.